pkg/hubble/parser/threefour/parser.go

```diff
@@ -183,7 +184,8 @@ func (p *Parser) Decode(data []byte, decoded *pb.Flow) error {
 	decoded.SourceNames = p.resolveNames(dstEndpoint.ID, srcIP)
 	decoded.DestinationNames = p.resolveNames(srcEndpoint.ID, dstIP)
 	decoded.L7 = nil
-	decoded.Reply = decodeIsReply(tn)
+	decoded.IsReply = decodeIsReply(tn)
+	decoded.Reply = decoded.GetIsReply().GetValue()
```

Comment:
do we need to check if `decoded.GetIsReply()` is nil? looks like `decodeIsReply` can return nil here.

Reply:
No. The protobuf getters take care of that :)
https://github.com/cilium/cilium/blob/8909c7f87a0c5d2b2e11e92b4c4e8ba37c60a03a/vendor/google.golang.org/protobuf/types/known/wrapperspb/wrappers.pb.go#L438-L443

Reply:
I think it's fine because this would only be if the flow (ie `decoded` here) is itself `nil`. But we should maybe check if `decoded` is `nil` early on.

Reply:
`decoded` being `nil` is something would panic much earlier. It currently will not happen, because none of the code is invoking the three/seven parsers directly (and parser.New always `decoded`).  

We could always add additional checks, but I'm marking this as resolved as it is completely unrelated to the PR.